From d82da8ada93219c0a575fed7e2869a3eaf96f123 Mon Sep 17 00:00:00 2001
From: Massimiliano Torromeo <massimiliano.torromeo@gmail.com>
Date: Wed, 26 Aug 2020 16:06:30 +0200
Subject: [PATCH] Meson remove mount version check

---
 meson.build | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 04331dd..eb72d99 100644
--- a/meson.build
+++ b/meson.build
@@ -792,8 +792,7 @@ if not libcap.found()
         libcap = cc.find_library('cap')
 endif
 
-libmount = dependency('mount',
-                      version : fuzzer_build ? '>= 0' : '>= 2.30')
+libmount = dependency('mount')
 
 want_seccomp = get_option('seccomp')
 if want_seccomp != 'false' and not fuzzer_build
-- 
2.28.0

